funnel/forms/venue.py

```diff
@@ -41,3 +43,12 @@ class VenueRoomForm(forms.Form):
     def validate_bgcolor(self, field):
         if not valid_color_re.match(field.data):
             raise forms.ValidationError("Please enter a valid color code")
+
+
+class VenuePrimaryForm(forms.Form):
+    venue = QuerySelectField(__("Venue"), validators=[forms.validators.DataRequired()],
+        get_pk=lambda v: v.suuid, get_label='title', allow_blank=False,
+        widget_attrs={'autocorrect': 'none', 'autocapitalize': 'none'})
+
+    def set_queries(self):
+        self.venue.query = Venue.query.filter_by(project=self.edit_parent)
```

Comment:
I'm uncomfortable with putting queries in forms because it will be hard to track down later when we need it. Try changing this to `self.venue.query = self.edit_parent.venues` and set `lazy='dynamic'` on that relationship.